main/res/values/strings.xml

```diff
@@ -812,6 +812,7 @@
     <string name="map_source_osm_osmde">OpenStreetMap.de</string>
     <string name="map_source_osm_cyclosm">CyclOSM</string>
     <string name="map_source_osm_offline">Offline</string>
+    <string name="map_source_osm_offline_attribution_pleasewait">(please wait...)</string>
```

Comment:
You should use the ellipsis character instead of dots and start with a captial letter "(Please wait…)"

Reply:
done and done